modules/storage/variables.tf

```
@@ -0,0 +1,9 @@
+variable "location" {
+  description = "The Azure region where the storage account will be created."
+  type        = string
+}
+
+variable "resource_group_name" {
+  description = "The name of the resource group in which to create the storage account."
+  type        = string
+}
```

Suggestion (optional): Add an optional `storage_account_name` variable (default null) so callers can override the generated name when needed. Right now the module generates a name internally using `random_string`, which is fine, but an override improves flexibility (helpful for testing or if a specific name is required). Also consider adding `validation` blocks for `location` and `resource_group_name` to fail fast on misconfiguration.